config/checkstyle.xml

```diff
@@ -109,6 +115,14 @@
       <property name="severity" value="error"/>
     </module>
 
+    <!-- Specific to WordPress -->
+    <module name="RegexpSingleline">
```

Comment:
This one's going to cause some trouble in the Woo app since it's not migrated to `WPSnackbar` yet (I'm not certain whether or not it will). It's probably okay to just leave this rule out when we fetch these changes in WooAndroid (or include the rule but comment it out with a note).

Reply:
I had a look at the checkstyle documentation and at the gradle checkstyle plugin doc. I couldn't find anything great to manage multiple configuration files. But Gradle would let us define multiple checkstyle tasks. We could use this to have multiple configuration files (one generic stored in this repo and one specific by project).

Might be tricky though... I found an example there: https://github.com/natgaertner/checkstylegradleparallelbug/blob/master/build.gradle